src/Uno.UI/UI/Xaml/ApplicationActivity.Android.cs

```diff
@@ -100,42 +99,57 @@ protected override void InitializeComponent()
 		public override bool DispatchKeyEvent(KeyEvent e)
 		{
 			var handled = false;
-			if (Uno.WinRTFeatureConfiguration.Focus.EnableExperimentalKeyboardFocus)
```

Comment:
Is EnableExperimentalKeyboardFocus no longer used on Android? Should we remove the property and adjust the docs?

Reply:
Yes, the property will no longer be relevant and keyboard focus will always be enabled. I have updated the docs, but kept the flag there to avoid breaking change, which will be done in 5x branch

Reply:
Since `master` is now 5x, should we remove the property from Android in this PR?